src/screens/SearchResult.js

```diff
@@ -42,7 +43,7 @@ const SearchResult = () => {
     }
   }, [loading]);
 
-  return loading ? <CircularProgress /> : displayCatalogs();
+  return loading ? <CircularProgress className="searchResult-loadingScreen" style={{width:"105em",height:"5em"}}/> : displayCatalogs();
```

Comment:
We added Inline css because CircularProgress has inline css and we want to override it and external CSS does not work in this case.

Reply:
use % - or align tags to get it in center